samples/ThreadedCommentExample/Program.cs

```diff
@@ -51,67 +51,65 @@ private static void Main(string[] args)
             }
 
             // WORKBOOK
-            SpreadsheetDocument? sd = ExampleUtilities.CreateSpreadsheetWorkbook(filePath, sheetName);
-
-            if (sd != null)
+            using (SpreadsheetDocument? sd = ExampleUtilities.CreateSpreadsheetWorkbook(filePath, sheetName))
```

Comment:
use `using SpreadsheetDocument? sd = ....;` instead without indenting the files

Reply:
Sure